### PR TITLE
Fit videos inside screen

### PR DESCRIPTION
**Pull request by MuriloucoLouco**

### CODE REVIEW

**Comment by RobTheFiveNine** [comment]
Hey! Thanks for the PR.

I've just given it a try, but it looks like the problem is still there. If you have a look at the screenshot below, you can see that the minimum height still seems to be determined by the video content itself, requiring you to scroll down:

image

Additionally, the video no longer uses 100% of the page width. Ideally, the solution will stretch the video player to 100% of the width, but not allow it to exceed the vertical bounds of the visible area.

**Comment by MuriloucoLouco** [comment]
Ok, i'll fix it.

File: app/index.html [hunk 1]
```diff
@@ -29,11 +29,14 @@
       }
 
       .video-container.visible {
+        height: 100vh;
```
Comment (RobTheFiveNine) [inline app/index.html:32]: Could you drop this bit? As long as we have the max height specified, we don't need an explicit height, specifying one breaks the layout a bit. Looks good without it though :smiley:

File: app/index.html [hunk 2]
```diff
@@ -29,11 +29,14 @@
...
         max-height: 100%;
+		max-width: 100%;
```
Comment (RobTheFiveNine) [inline app/index.html:34]: This can be dropped too - no need for the `max-width` in this case

File: app/index.html [hunk 3]
```diff
@@ -29,11 +29,14 @@
...
       }
-	  
+
       video {
-        width: auto;
+        max-height: 100vh;
+        max-width: 100vw;
```
Comment (RobTheFiveNine) [inline app/index.html:39]: This can be dropped too - we only need the `max-height`

**Review by RobTheFiveNine — CHANGES_REQUESTED** [review]

File: app/index.html (hunk 3 above)
Comment (RobTheFiveNine) [reply app/index.html:39]: For clarity - I meant drop the `width` and `max-width` properties :smiley:

**Comment by MuriloucoLouco** [comment]
Removed `max-width` and `height` properties. Everything is right now, i think.

**Review by RobTheFiveNine — APPROVED** [review]

**Comment by RobTheFiveNine** [comment]
Thanks @MuriloucoLouco - just gave it a test and it works great :tada:

**Pull request merged**